remove_employee_webhook_receiver.py

```diff
@@ -0,0 +1,52 @@
+import traceback
+import json
+import requests
+
+from ssm_util import fetch_ssm_params
+
+# Removes an employee from the guide
+# once their employment has been terminated
+
+
+def lambda_handler(event, context):
+    try:
+        # Fetch the Builder API key, the guide ID of the guide where the content
+        # is published, and the custom list ID that the items are associated with
+        api_key, guide_id, employee_customlist_id, zenefits_app_key = fetch_ssm_params()
+
+        # Initialize a Session
+        session = requests.Session()
+        session.headers.update({"Authorization": f"JWT {api_key}"})
+
+        data = event["data"]
+
+        # Fetch the existing CustomListItem from Builder by filtering on the import_id field.
+        # This is needed to obtain the CustomListItem.id, which is required in the PATCH request
+        url = "https://builder.guidebook.com/open-api/v1/custom-list-items/?guide={}&custom_lists={}&import_id={}".format(
+            guide_id, employee_customlist_id, data["id"]
+        )
+        response = session.get(url)
+        response.raise_for_status()
+        custom_list_items = response.json()["results"]
+
+        # If there is more than one matching CustomListItem, delete them all
+        for custom_list_item in custom_list_items:
+            url = "https://builder.guidebook.com/open-api/v1/custom-list-items/{}/".format(
```

Comment:
style nitpick: let's write this as a formatter string to be consistent with the rest of the project